quantum-viz/quantum_viz/utils.py

```diff
@@ -0,0 +1,85 @@
+import json
+import tempfile
+import warnings
+import webbrowser
+from pathlib import Path
+from typing import Any
+from typing import Dict
+from typing import Optional
+from typing import TYPE_CHECKING
+from typing import Union
+
+if TYPE_CHECKING:
+    from qiskit import QuantumCircuit
+
+HTML_TEMPLATE = """<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>quantum-viz.js visualization</title>
+</head>
+<body>
+  <div id="msg" style="font-size: large; font-weight: bolder;">
+    There was some error with the script
+  </div>
+  <div id="circuit_div"></div>
+  <script src="https://unpkg.com/@microsoft/quantum-viz.js{0}"></script>
```

Comment:
We should just use the same HTML in the Viewer. You can do this by creating an instance of Viewer and call the `html_str` method.

Reply:
The [html template](https://github.com/microsoft/quantum-viz.js/blob/bed20253f6333aca10ab465ad71409901efedac1/quantum-viz/quantum_viz/widget.py#L21) there does not have all the fields from here (specifically version), and to my understanding is not even a complete HTML (no body).
It may be possible to somehow use the sub-html template from there, but I prefer keeping it to another PR.

Reply:
That's fine. Paying close attention, turns out they are different because they are used in different context, even though they are for the most part doing the same.